### ISSUE TITLE
Log file?

### ISSUE THREAD

**Issue by theonetmat** [andrewshilliday/garage-door-controller#29]
Is there a log file or other place where I can see the history of the door actions? 

Thanks for the project. It works great so far.

**Comment by atljam** [comment]
cat /var/log/syslog | grep garage_controller
make sure you use the pipe symbol its shift backslash key

**Comment by theonetmat** [comment]
Thanks for the idea. I kept looking and did find the syslog and was able to get something similar working. I tried to use a tail-f and then have the grep write to another log file. That does work but only once. What I am doing so far is what I have below. The works but I have to run it manually. I'll eventually have it run maybe once a day to append the new log file. I am not sure what will happen to the file size over time if I keep having it add to the same file.

grep --line-buffered "garage_controller" syslog  >> garagedoorlog.txt
